shapes/go-cam-shapes.shex

```diff
@@ -59,17 +59,29 @@ PREFIX causally_upstream_of_positive_effect: <http://purl.obolibrary.org/obo/RO_
   rdfs:comment: . *;
 } // rdfs:comment  "Default allowable metadata for GO-CAM entities"
 
+<OwlClass> {
+  rdf:type [ owl:Class ] {1} ;
+}
+
+<BiologicalProcessClass> @<OwlClass> AND {
+  rdfs:subClassOf [ GoBiologicalProcess: ] ;
+}
+
 ##something wrong with this one
 # note it effects other shapes that use it
-<BiologicalProcess> @<GoCamEntity> AND {
-  bl:category [GoBiologicalProcess:]  ;
+<BiologicalProcess> @<GoCamEntity> AND EXTRA a {
+  a @<BiologicalProcessClass> {1};
```

Comment:
Is `{1}` not automatic? Or is that needed because of `EXTRA`?

Reply:
I like being explicit but maybe it's cleaner to leave off